models/adminModel.js

```diff
@@ -13,6 +17,26 @@ const adminSchema = mongoose.Schema({
     type: String,
     required: true
   }
+  phone: {
+    type: Number,
+    required: true
+  },
+  sellerName: {
+    type: String,
+    required: true
+  },
+  sellerAddress: {
+    type: String,
+    required: true
+  },
+  regNumber: {
+    type: String,
+    required: true
+  },
+  scale: {
+    type: String,
+    required: false
+  }
```

Comment:
Remove all these fields. We are making an Ecommerce website for a particular store only. So no need for sellers Adress, regn number and all that stuff.